lua/barbar/buffer.lua

```diff
@@ -90,10 +86,15 @@ function buffer.get_name(buffer_number, depth)
   local hide_extensions = config.options.hide.extensions
 
   if name ~= '' then
-    local full_name = buf_get_option(buffer_number, 'buftype') == 'terminal' and
-      terminalname(name) or (hide_extensions and fnamemodify(name, ':t') or name)
```

Comment:
I don't know what my obsession was with one-liners was a few years back. This was almost impossible to read, I rewrote it as an `if`.